client/src/main/java/com/alibaba/nacos/client/naming/net/NamingProxy.java

```diff
@@ -513,7 +519,20 @@ public String reqApi(String api, Map<String, String> params, Map<String, String>
         }
         
         NacosException exception = new NacosException();
-        
+
+        if (StringUtils.isNotBlank(nacosDomain)) {
+            for (int i = 0; i < UtilAndComs.REQUEST_DOMAIN_RETRY_COUNT; i++) {
```

Comment:
Where is the place `maxRetry ` used?  The count limit should be `maxRetry`

Reply:
my fault, fixed.